README.md

```diff
@@ -0,0 +1,17 @@
+# Service Access Layer
+
+Provides tools to generate from XML description language and database bindings
+for LSST commands, events and telemetry. Depends on ts\_openslice.
+
+# Usage
+
+To get needed github projects and setup environment variables, do:
+
+```bash
+git clone https://github.com/lsst-ts/ts_opensplice
+git clone https://github.com/lsst-ts/ts_sal
+source ts_sal/setup.env
+```
+
+After that, you shall be able to call salgenerator and see it producing a help
+text. Please see [test README](test/README.md) for details.
```

Comment:
Please provide the `salgenerator` command that prints help right here, rather than referring users elsewhere for that. Referring the user elsewhere for additional details about using salgenerator seems quite appropriate (though I would not expect that location to be test/README.md).